packages/assistify-ai/server/methods/SmartiReload.js

```diff
@@ -137,7 +140,9 @@ Meteor.methods({
 			conversation.messages.push(newMessage);
 		}
 
-		SmartiProxy.propagateToSmarti(verbs.post, 'conversation', conversation);
+		const response = SmartiProxy.propagateToSmarti(verbs.post, 'conversation', conversation);
+		SystemLogger.debug('Update conversation with id: ', response.id);
+		SmartiAdapter.updateMapping(rid, response.id);
```

Comment:
I believe this should be done implicitly inside the Smarti adapter. Actually, I would expect we don't use the `SmartiProxy` outside the Smarti Adapter. I obviously oversaw this earlier on.